### PR TITLE
feat(elixir): add queries

### PR DESCRIPTION
**Pull request by mhanberg**

### CODE REVIEW

File: queries/elixir/context.scm [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+((unary_operator
+   operand: (call
+      target: (identifier)
+      (arguments (_)))) @_op (#lua-match? @_op "@[%w_]+")) @context
+
+(stab_clause right: (body (_) @context.end)) @context
+
+(call
+  target: (identifier) @_ident
```
Comment (lewis6991) [inline queries/elixir/context.scm:17]: You don't need the `@_ident` capture.

**Comment by lewis6991** [comment]
Can you also add some examples in the `test/` directory?

File: queries/elixir/context.scm (hunk 1 above)
Comment (mhanberg) [reply queries/elixir/context.scm:17]: Thanks, leftover from an earlier iteration of the query.

**Comment by mhanberg** [comment]
> Can you also add some examples in the `test/` directory?

Yes

**Comment by mhanberg** [comment]
@lewis6991 all done

**Comment by lewis6991** [comment]
Thanks!

**Pull request merged**